WrathCombo/Combos/PvE/GNB/GNB.cs

```diff
@@ -677,47 +679,49 @@ protected override uint Invoke(uint actionID, uint lastComboMove, float comboTim
+                                return All.Rampart;
+                        }
+
                         //Superbolide
                         if (IsEnabled(CustomComboPreset.GNB_ST_Superbolide) && //Superbolide option is enabled
                             ActionReady(Superbolide) && //Superbolide is ready
                             PlayerHealthPercentageHp() < Config.GNB_ST_Superbolide_Health && //Player's health is below selected threshold
-                            (Config.GNB_ST_Superbolide_SubOption == 1 || //Superbolide is enabled for all targets
-                            (TargetIsBoss() && Config.GNB_ST_Superbolide_SubOption == 2))) //Superbolide is enabled for bosses only
+                            (Config.GNB_ST_Superbolide_SubOption == 0 || //Superbolide is enabled for all targets
+                            (TargetIsBoss() && Config.GNB_ST_Superbolide_SubOption == 1))) //Superbolide is enabled for bosses only
                             return Superbolide;
 
-                        //Nebula
-                        if (IsEnabled(CustomComboPreset.GNB_ST_Nebula) && //Nebula option is enabled
-                            ActionReady(OriginalHook(Nebula)) && //Nebula is ready
-                            PlayerHealthPercentageHp() < Config.GNB_ST_Nebula_Health && //Player's health is below selected threshold
-                            (Config.GNB_ST_Nebula_SubOption == 1 || //Nebula is enabled for all targets
-                            (TargetIsBoss() && Config.GNB_ST_Nebula_SubOption == 2))) //Nebula is enabled for bosses only
-                            return OriginalHook(Nebula);
-
-                        //Rampart
-                        if (IsEnabled(CustomComboPreset.GNB_ST_Rampart) && //Rampart option is enabled
-                            ActionReady(All.Rampart) && //Rampart is ready
-                            PlayerHealthPercentageHp() < Config.GNB_ST_Rampart_Health && //Player's health is below selected threshold
-                            (Config.GNB_ST_Rampart_SubOption == 1 || //Rampart is enabled for all targets
-                            (TargetIsBoss() && Config.GNB_ST_Rampart_SubOption == 2))) //Rampart is enabled for bosses only
-                            return All.Rampart;
-
                         //Camouflage
                         if (IsEnabled(CustomComboPreset.GNB_ST_Camouflage) && //Camouflage option is enabled
                             ActionReady(Camouflage) && //Camouflage is ready
                             PlayerHealthPercentageHp() < Config.GNB_ST_Camouflage_Health && //Player's health is below selected threshold
-                            (Config.GNB_ST_Camouflage_SubOption == 1 || //Camouflage is enabled for all targets
-                            (TargetIsBoss() && Config.GNB_ST_Camouflage_SubOption == 2))) //Camouflage is enabled for bosses only
```

Comment:
This will apply to all the other similar changes, but without resetting the config you'll have users stuck if they selected the old option 2. All you need to do is change the config key and it'll reset it.